src/freecad/cycax_part_freecad.py

```diff
@@ -0,0 +1,631 @@
+#
+# This file is called directly from FreeCAD.
```

Comment:
I think this should rather be in a file docstring. This would mean """info""" over #info.

Reply:
updated. it is in the follow up PR.